scripts/ts-bin/csvtojsonlines_test.ts

```diff
@@ -0,0 +1,34 @@
+import {csvtojsonlines} from './csvtojsonlines_lib';
+import * as stream from 'stream';
+import {expect} from 'chai';
+import 'mocha';
+
+describe('Hello function', () => {
+  it('should return hello world', async function () {
```

Comment:
Did you mean to rename the describe/it names? (Though I have no objection to a friendly greeting from a test!)

Reply:
oops, thanks! fixed!